python/semantic_kernel/connectors/memory/azure_ai_search/azure_ai_search_memory_store.py

```diff
@@ -32,7 +32,7 @@
 logger: logging.Logger = logging.getLogger(__name__)
 
 
-class AzureCognitiveSearchMemoryStore(MemoryStoreBase):
+class AzureAiSearchMemoryStore(MemoryStoreBase):
```

Comment:
Could we also please update the AzureAISearch package to the latest if we aren't using it already?